README.md

```
@@ -1,194 +1,379 @@
+<string name="appupdater_or">or</string>
+<string name="appupdater_download_from_store">Download from store</string>
+<string name="appupdater_couldnt_find_downloaded_file">Couldn't find downloaded file</string>
+```
+
+---
+
+## Architecture
+
+The library is split into focused modules:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)